BPOR/BPOR.Domain/Migrations/20241119153650_CRNCC2416-notify-letter.cs

```diff
@@ -0,0 +1,294 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+#pragma warning disable CA1814 // Prefer jagged arrays over multidimensional
+
+namespace Dynamo.Stream.Handler.Migrations
+{
+    /// <inheritdoc />
+    public partial class CRNCC2416notifyletter : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropTable(
+                name: "EmailCampaignParticipants");
+
+            migrationBuilder.DropTable(
+                name: "EmailCampaigns");
+
+            migrationBuilder.DropTable(
```

Comment:
will dropping these tables first lead to any lost data?

Reply:
i was thinking of removing the migrations, as we all have overlaps. we can then do one migration for this work once we merge it all? and ensure we aren't dropping tables or columns like this